frontend/src/apiAccess/selectors/activityLogSelectors.js

```diff
@@ -37,11 +37,12 @@ export const getActivityLogCountsByDayForSingleUser = ({
   const resultByDate = {};
 
   eventTypes.forEach((eventType) => {
-    const userLogsFilteredByType = userLogs.filter(
-      (element) =>
-        element.id.match(`event_type__name=${eventType}&`) ||
-        element.id.match(`event_type__name=${eventType}$`)
-    );
+    const userLogsFilteredByType = userLogs.map((type) => {
```

Comment:
This code was confusing. Why did you choose to name a logEntry a type?

If we have an array of apples and we want to do a map on the apples then the code would look like this

```
apples = [apple1,apple2, apple3]
apples.map( (apple, index)=>{whatever})
//                      ^^^^
//                     every element is an apple
```

If we are looping over log entries then every element is not a type. It is a log entry.

If you were a carpenter then would you call a nail a hammer? No, it would confuse people. Call things what they are.